extensions/wikia/IndexingPipeline/PipelineEventProducer.class.php

```diff
@@ -55,9 +55,12 @@ public static function onNewRevisionFromEditComplete( $article, Revision $rev, $
 		$ns = self::preparePageNamespaceName( $article->getTitle() );
 		$action = $rev->getPrevious() === null ? self::ACTION_CREATE : self::ACTION_UPDATE;
 		$id = $article->getId();
+		$data = [
+			'revisionId' => $rev->getId()
```

Comment:
Can't this be acquired from id in pipeline push? It could be more reliable and implemented only once for all messages.